### PR TITLE
New Component: mate-settings-daemon

### PR DESCRIPTION
**Pull request by Toasterson**

### CODE REVIEW

**Comment by pyhalov** [comment]
The same question about gnome-settings-daemon patches. Have you looked through them?

**Comment by pyhalov** [comment]
At least components/desktop/gnome2/gnome-settings-daemon/patches/gnome-settings-daemon-14-libXi.patch should be added!

**Comment by Toasterson** [comment]
Not many patches still apply. Mate renamed all occurences of gnome and gsd in the code to mate and msd respectively plaus massively refactored.

**Comment by pyhalov** [comment]
gnome-settings-daemon-13-exit-action.patch apply (at least can be modified to apply), the question is what does it change? :)

File: components/desktop/mate/mate-settings-daemon/patches/gnome-settings-daemon-06-xft.patch [hunk 1]
```diff
@@ -0,0 +1,33 @@
+diff -up mate-settings-daemon-1.14.0/config.h.in.orig mate-settings-daemon-1.14.0/config.h.in
```
Comment (pyhalov) [inline components/desktop/mate/mate-settings-daemon/patches/gnome-settings-daemon-06-xft.patch]: this one seems to be useless
I don't see that HAVE_XFT2 is used

**Comment by Toasterson** [comment]
gnome-settings-daemon-13-exit-action.patch only changes that the logout dialog is displayed instead of the powerof dialog when pressing the shutdown button. Thats a change that is logical on a server not that much on a desktop where the user should be able to shutdown the computer.

File: components/desktop/mate/mate-settings-daemon/patches/gnome-settings-daemon-06-xft.patch (hunk 1 above)
Comment (Toasterson) [inline components/desktop/mate/mate-settings-daemon/patches/gnome-settings-daemon-06-xft.patch]: Upon closer inspection it looks like pango provides the same things as xft2 does. so we don't need this patch.

**Pull request merged**